src/main/java/com/byznass/tiolktrack/kernel/model/User.java

```diff
@@ -10,7 +10,7 @@
 	private final byte[] passHash;
 	private final byte[] passSalt;
 
-	public User(@NotNull String id, @NotNull byte[] passHash, @NotNull byte[] passSalt) {
+	public User(@NotNull String id, @NotNull byte[] passHash, @NotNull byte[] passSalt) throws RuntimeException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3AS00112)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.RuntimeException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)